documentation/assemblies/upgrading/assembly-upgrade.adoc

```diff
@@ -6,10 +6,40 @@
 [id='assembly-upgrade-{context}']
 = Upgrading Strimzi
 
-Strimzi can be upgraded from a lower version to a higher version to take advantage of new features and enhancements, performance improvements, and security options.
+Strimzi can be upgraded to version {ProductVersion} to take advantage of new features and enhancements, performance improvements, and security options.
 
-As part of the upgrade, you upgrade Kafka to a higher supported version. 
-Each Kafka release introduces new features, improvements, and bug fixes.
+As part of the upgrade, you upgrade Kafka to the latest supported version.  
+Each Kafka release introduces new features, improvements, and bug fixes to your Strimzi deployment.
+
+Strimzi can be xref:assembly-downgrade-{context}[downgraded] to the previous version if you encounter issues with the newer version.
+
+.Upgrade paths
+
+Two upgrade paths are possible:
+
+Incremental:: 
+Upgrading Strimzi from version 0.21.1 to version {ProductVersion}.
```

Comment:
We should write around the 0.21.1 release otherwise we would need to update it all the time. MAybe ...

```suggestion
Upgrading Strimzi from previous version to version {ProductVersion}.
```

Reply:
OK, I guess it's likely to get missed. I'll change it to "the previous version".

Reply:
But does previous version mean 0.20.0 or a 0.20.1? It's a bit ambiguous. Do we mean previous _minor_ version?

Reply:
I guess that is good point @laidan6000 ^^^ ... _previous minor version_ would be better.

Reply:
Maybe an _(X.Y)_ example in

Reply:
Changed to "the previous minor version". I tried an X.Y example but it seemed confusing.